io:
  # Incoming point cloud topic.
  scanner_topic: /velodyne_points #/mapper/cloud

  # Filtered scan output topic.
  filtered_topic: filtered

  # Unfiltered scan output topic.
  unfiltered_topic: unfiltered

  # World frame.
  world_frame: world

  # Odometry frame.
  odometry_frame: odom

  # Localized frame.
  localized_frame: slam